pkg/apis/pingcap/v1alpha1/pd_config.go

```diff
@@ -139,7 +144,26 @@ type DashboardConfig struct {
 	// When not disabled, usage data will be sent to PingCAP for improving user experience.
 	// Optional: Defaults to false
 	// +optional
+	//+k8s:openapi-gen=false
```

Comment:
add a deprecated notice in the comment, g.g. `Deprecated in PD v4.x.x, use EnableTelemetry instead`

please update the `v4.x.x` to the correct version in which this field is deprecated.

Reply:
If the PD plans to remove this field in the future, please add the removal version too. For example, `Deprecated in PD v4.x.x, planned for removal in vx.y.z`

Reply:
deleted in https://github.com/pingcap/tidb-operator/pull/2964/commits/46c14c678a2bf9492f21a6b34e8f26b0d0f75822